kairon/shared/chat/processor.py

```
@@ -235,3 +239,71 @@
         comment_response = channel.get("config", {}).get("static_comment_reply")
         return comment_response
 
+    @staticmethod
+    async def validate_and_save_media_file(bot: Text, user: Text, file_content: File):
```

Ruff (0.12.2)

Unused static method argument: `user`

(ARG004)